### PR TITLE
openj9 0.32.0

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

### CODE REVIEW

File: Formula/openj9.rb [hunk 1]
```diff
@@ -72,7 +72,7 @@ class Openj9 < Formula
 
   resource "omr" do
     url "https://github.com/eclipse-openj9/openj9-omr.git",
-        tag:      "openj9-0.30.1",
+        tag:      "openj9-0.32.0",
```
Comment (Porkepix) [inline Formula/openj9.rb:75]: I don't get how it can update tag while not touching the revision?!

**Comment by Porkepix** [comment]
Not sure one of the resources tag is moved while the other isn't, nor why the moved tag doesn't have its revision updated? Also, despite the link, where are picked exactly the `boot-jdk` urls?

File: Formula/openj9.rb (hunk 1 above)
Comment (cho-m) [reply Formula/openj9.rb:75]: Bump doesn't support auto-updating resources; however, the find-and-replace behavior will modify any matching version pattern. Will need to manually update this.

**Comment by cho-m** [comment]
The book JDK is the same for now. To check:
- From https://github.com/eclipse-openj9/openj9/blob/openj9-0.32.0/doc/build-instructions/, you want latest Java version available (currently 17)
- In https://github.com/eclipse-openj9/openj9/blob/openj9-0.32.0/doc/build-instructions/Build_Instructions_V17.md, you can search for hyperlinks on boot JDK or adoptopenjdk:
  - Linux: https://api.adoptopenjdk.net/v3/binary/latest/16/ga/linux/x64/jdk/openj9/normal/adoptopenjdk
  - macOS: https://api.adoptopenjdk.net/v3/binary/latest/16/ga/mac/x64/jdk/openj9/normal/adoptopenjdk
- Then follow redirect to final URL. In this case, they haven't changed.

Need to manually update the revision number for `omr`.

`openj9-openjdk-jdk` will require updating to branch/commit for https://github.com/ibmruntimes/openj9-openjdk-jdk17/tree/v0.32.0-release

**Comment by Porkepix** [comment]
> The book JDK is the same for now. To check:
> 
>     * From [https://github.com/eclipse-openj9/openj9/blob/openj9-0.32.0/doc/build-instructions/](https://github.com/eclipse-openj9/openj9/blob/openj9-0.32.0/doc/build-instructions/?rgh-link-date=2022-04-23T05%3A36%3A54Z), you want latest Java version available (currently 17)

Isn't it 18 now?

**Comment by cho-m** [comment]
Although they haven't added build instructions in the tag, you can give JDK 18 a try as it is in release notes. Instructions are available in master branch https://github.com/eclipse-openj9/openj9/blob/master/doc/build-instructions/Build_Instructions_V18.md

In which case, the book JDK will change (it will need JDK17), and you will need to use https://github.com/ibmruntimes/openj9-openjdk-jdk18.git

**Comment by Porkepix** [comment]
Okay, tried an update to resources while keeping it for openjdk17 as 18 isn't in this release yet.
I'm unsure of what I did put though: I was unable to find where was the previous commit id for `openj9-openjdk-jdk`, therefore unsure of the new one I picked.

Also, I'm quite surprised it's needed to provide both a branch and a commit id: shouldn't a commit id alone be enough? Or a tag?

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**